econplayground/templates/assignment/step_detail.html

```diff
@@ -102,75 +102,111 @@ <h2>{{object.question.title}}</h2>
                         {% endfor %}
                     {% endfor %}
                 </div>
-
-                {% if submission != None %}
-                    {% if object.question.first_rule.has_feedback %}
-                        <div class="alert alert-light" role="alert">
-                            {% if submission and object.question.first_rule.has_fulfilled_feedback %}
-                                <p>
-                                    {{object.question.first_rule.feedback_fulfilled}}
-                                </p>
-
-                                {% if object.question.first_rule.media_fulfilled %}
-                                    <img src="{{object.question.first_rule.media_fulfilled.url }}"
-                                         class="img-fluid" alt="Question fulfilled media"/>
-                                {% endif %}
-                            {% elif object.question.first_rule.has_unfulfilled_feedback %}
-                                <p>
-                                    {{object.question.first_rule.feedback_unfulfilled}}
-                                </p>
-
-                                {% if object.question.first_rule.media_unfulfilled %}
-                                    <img src="{{object.question.first_rule.media_unfulfilled.url }}"
-                                         class="img-fluid" alt="Question unfulfilled media"/>
+                {% if multiple_choice|length > 0 or has_assessment%}
```

Comment:
`has_assessment` should encompass the presence of multiple choice - a multiple choice quiz is just another form of assessment.